mackup/applications/vs4mac.cfg

```diff
@@ -0,0 +1,7 @@
+[application]
+name = Visual Studio for Mac
+
+[configuration_files]
+Library/VisualStudio
```

Comment:
Which files are saved in this folder? Just config or binary stuff?

Reply:
Just configs, really similar to Xamarin Studio: color themes, key bindings, style policies, snippets. All JSON and XML.